src/test/java/org/javarosa/core/model/RepeatTest.java

```diff
@@ -0,0 +1,37 @@
+package org.javarosa.core.model;
+
+import org.javarosa.core.test.Scenario;
+import org.javarosa.form.api.FormEntryController;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.javarosa.core.util.BindBuilderXFormsElement.bind;
+import static org.javarosa.core.util.XFormsElement.*;
+import static org.junit.Assert.assertThat;
+
+public class RepeatTest {
+
+    @Test
+    public void whenRepeatIsNotRelevant_isSkipped() throws Exception {
```

Comment:
I think it'd be helpful for the name to be a little more specific. Maybe just adding `repeatPromptIsSkipped`.